pydata/checks.py

```diff
@@ -0,0 +1,46 @@
+from django.apps import apps as camelot_apps
+from django.core.checks import register, Error, Tags
+
+from amy.urls import urlpatterns
+from workshops import urls as workshops_urls
+
+from . import urls as pydata_urls
+
+
+@register(Tags.urls)
```

Comment:
Interesting idea. It didn't occur to me we could use this subsystem. BTW I think @chrismedrela worked on it in some previous GSOC :-)